### PR TITLE
Contact Section

### PR DESCRIPTION
**Pull request by Salarhabibi-amir**
I have Created the contact section trough pair programming, we used flex to place the elements. and I used the HTML5 features for validation.

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi Team @Salarhabibi-amir and @AndreaM2429 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing Linters. Kindly make sure all the linters are passing before you can submit again

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -281,7 +282,39 @@ <h2>Skills</h2>
...
+            <div class="contact-items">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="form-flex">
+                    <div class="form-flex-item ">
+                        <input type="text" placeholder="Lorem Ipsum" maxlength="30" name="userName">
+                    </div>
+                    <div class="form-flex-item ">
+                        <input type="emial" placeholder="loremipsum@gmail.com" name="emial">
+                    </div>
+                    <div class="form-flex-item textarea">
+                        <textarea name="" id="" cols="30" rows="12" maxlength="500" required name="text">Write your message here...
```
Comment (ademibowale) [inline index.html]: - [ ] when i submit the form it displayed form not foumd here the screenshot

form not sun

- [ ] kindly consider to  add reqired attribut to activate the form , be confirm that the form is submitted successfully with valid input values. All input fields have the required attribute to activate the validation on submit  After submitting the form it should display  comfirmation page that the form was submitted successfully.kindly fix this

File: index.html [hunk 2]
```diff
@@ -281,7 +282,39 @@ <h2>Skills</h2>
...
+            <div class="contact-items">
+                <h2>Contact me</h2>
+            </div>
+            <div class="contact-items">
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!</p>
 
+            </div>
+            <div class="contact-items">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="form-flex">
+                    <div class="form-flex-item ">
```
Comment (ademibowale) [inline index.html]: - [ ] I would kindly suggest you to wrapp All input fields  in a single `form` element.this will indicate a proper semantic html

**Review by ademibowale — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @Salarhabibi-amir , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight 🥇

- Descriptive Pull Request title and summary. ✔️
- Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). ✔️
- Max-length attribute Used on the input ✔️
- Pair-programming collaborative activity ✔️
- Good use of Github flow ✔️
- All linters checks passed ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -281,7 +282,39 @@ <h2>Skills</h2>
...
+            <div class="contact-items">
+                <form action="https://formspree.io/f/xeqwnkae" method="post" class="form-flex">
+                    <div class="form-flex-item ">
+                        <input type="text" placeholder="Lorem Ipsum" maxlength="30" name="userName">
+                    </div>
+                    <div class="form-flex-item ">
+                        <input type="emial" placeholder="loremipsum@gmail.com" name="emial">
+                    </div>
+                    <div class="form-flex-item textarea">
+                        <textarea name="" id="" cols="30" rows="12" maxlength="500" required name="text">Write your message here...
```
Comment (jmonto55) [inline index.html]: - [ ] [OPTIONAL] Great work including your `<textarea>` field here 💯. However, please kindly consider removing the text "write your message here..." from the tag and including that text as a placeholder property instead. 🤓 This change will enhance the user experience of your form. 🔧

File: style.css [hunk 4]
```diff
@@ -368,3 +368,92 @@ p {
...
+  border-radius: 8px;
+  padding: 24px;
+  color: #172b4d;
+  font-size: 17px;
+}
+
+.textarea {
+  background-color: white;
+  border-radius: 5px;
+}
```
Comment (jmonto55) [inline style.css:426]: - [ ] [OPTIONAL] Good work applying styles to your `<textarea>` element ✔️. However, I noticed the font color of your textarea is almost white, kindly consider changing the color to a darker one to enhance the user experience of your form. 🤓

    > Hint to style the textarea placeholder:
    ```
    .textarea::placeholder {
        color: #000;
    }
    ```

**Review by jmonto55 — APPROVED** [review]
# STATUS: APPROVED 🟢

Hi @Salarhabibi-amir, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

AgreeSealGIF


## Optional suggestions
- [ ] [OPTIONAL] Great work building your web. 👌However, I noticed your page has an overflow in the x axis in mobile. I think the problem is inside the class `.headline-container`.
    > Try removing the CSS property `min-width: 60vh;` from the `.headline-container` class.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**